NumLib/Fem/Deformation/BMatrixPolicy.h

```diff
@@ -61,6 +65,7 @@ struct KelvinMatrixTypeImpl<1>
     using type = Eigen::Matrix<double, 1, 1, Eigen::ColMajor>;
 };
 
+/// Kelvin vector type for given displacement dimension.
```

Comment:
I think the documentation is wrong. A KelvinMatrix is a 4th order tensor which is mapped to a square matrix under the Kelvin mapping.